cosmoz-data-nav.js

```diff
@@ -749,29 +754,31 @@
 						return idx;
 					}
 
-					const incomplete = this.isIncompleteFn(item);
-					this._toggleInstance(element, idx, incomplete);
-
-					if (incomplete) {
-						// no data for item, instance has been toggled
-						// drop task from queue
+					if (this.isIncompleteFn(item)) {
+						element.item = false;
+						// no data for item drop task from queue
 						return;
 					}
 
+					element.__incomplete._showHideChildren(true);
+
 					if (element.item === item) {
-						// already rendered
+						renderRun = idx === this.selected && this._notifyElementResize();
```

Comment:
Yes the PR should be merged after #21. Will change it so that `renderRun` will be  a Boolean. 
If `idx` is the selected index and we have done a element resize we should not do more in this run because the resize has been expensive.